PolishWithBlondes/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,39 @@
+import './navbar.scss';
+import logo from '../../assets/logo.png';
+import hamburger from '../../assets/hamburger.png';
+
+export const Navbar = () => {
+  return (
+    <nav className="navbar">
+      <img src={logo} className="logo" />
+      <ul>
+        <li>
+          <a href="">Book a lesson</a>
+        </li>{' '}
+        <li>
+          <a href="">Worksheets</a>
+        </li>{' '}
+        <li>
+          <a href="">Podcasts</a>
+        </li>{' '}
+        <li>
+          <a href="https://www.youtube.com/@polishwithblondes">Youtube</a>
+        </li>
+        <li>
+          <a href="https://buymeacoffee.com/polishwithblnds">
+            support our work
+          </a>
+        </li>
+        <li>
+          <a href="">About us</a>
+        </li>
+        <li>
+          <a href="">Contact</a>
+        </li>
+      </ul>
+      <div className="hamburger">
+        <img src={hamburger} />
```

Comment:
Later on, maybe we could use a Hamburger component from the npm

Reply:
Ok, let's do that later on